src/Migration.php

```diff
@@ -1716,6 +1716,26 @@ private function migrateSearchOptions()
                     'itemtype' => $itemtype,
                     'num'      => $old_search_opt
                 ]);
+
+                // Update template fields
+                $item = new $itemtype();
+                if ($item instanceof CommonITILObject) {
```

Comment:
Will prevent any error if the target itemtype is not a valid class.

```suggestion
                if (is_a($itemtype, 'CommonITILObject', true)) {
```